app/serializers/rating_info_serializer.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class RatingInfoSerializer < ActiveModel::Serializer
+  attribute :user_percent_of_disability
+
+  def id
+    nil
```

Comment:
The JSON output always has a null id? Is that because of the structure of JSON that's expected?

Reply:
JSON::API requirement